packages/core/src/tracing/spanHandle.js

```diff
@@ -150,6 +166,7 @@ SpanHandle.prototype.disableAutoEnd = function disableAutoEnd() {
 
 /**
  * Finishes as span that has been switched to manual-end-mode before.
+ * @param {boolean | number} errorCount
```

Comment:
*issue:* I think this should always be a number. Do we pass in a boolean somewhere? That might be wrong.

Reply:
Line 175 to 177:
```javascript
  if (errorCount === true) {
    errorCount = 1;
  }
```